routes/movie.js

```diff
@@ -0,0 +1,176 @@
+const express = require("express");
+const router = express.Router();
+const movies = require("../data/movies");
+const users = require("../data/users");
+const validation = require("../utils/validation");
+
+router.get("/:id", async (req, res) => {
```

Comment:
i created a movies (with an s) file as well that is completely useless lol. We can delete that from main?

Reply:
Yeah, sorry about that!